### ISSUE TITLE
Make command not found on Windows when compiling from the dosxyznrc GUI

### ISSUE THREAD

**Issue by ftessier** [nrc-cnrc/EGSnrc#53]
On Windows 10 (and presumably other versions of Windows), trying to compile `dosxyznrc` on from the GUI causes a fatal error: `couldn't execute "make": no such file or directory`. This is likely due to the fact that when using MinGW, the `make` command is in fact called `mingw32-make`. There should be a generic `make` command made available via the EGSnrc configuration on Windows.

**Comment by ftessier** [comment]
@mainegra says: In the config file `%EGS_CONFIG%` there is a variable called `%make_prog%` which should be used when invoking make from any GUI.

**Comment by crcrewso** [comment]
May I suggest that we add a documentation change to the install guide explicitly listing installing the official version of make for windows (or cmake) with notes about how to add each to the path. It's a lot easier than ming/cygwin for people who have not seen these systems before.

**Comment by ftessier** [comment]
We could if it works. The problem is that as far as I remember some EGSnrc makefiles rely in part on specific `GNU make` extensions. Unless you can confirm that they work with other versions of make with minimal modifications (and then again soon we'll be using GNU make on WSL!)

**Comment by crcrewso** [comment]
Here's the version I found which appears to be some good samaritan compiling the official code to a windows executable. I copied it to the Program Files/make directory and added the location to my path. [http://www.equation.com/servlet/equation.cmd?fa=make](url)

I spoke too soon about cmake, That one was only tested on the WSL and being the Ubuntu version might have had some compatability flag turned on without me knowing.
